src/main/java/org/prnhs/javaee/swim/services/ContactsServices.java

```diff
@@ -66,14 +82,17 @@ public ContactsDto getById(Integer id){
                 dtos.add(dto);
             }
         }
-        
+
         return dtos;
     }
-    
+
     public void delete(Integer id) {
         Contacts contacts = dao.findOne(id);
+        LOGGER.debug("The delete method is called");

```

Comment:
do this as the first thing... kind of letting the other developer know that you entered this method with so and so parameter.  And you do teh same when exiting the method.